pom.xml

```diff
@@ -76,6 +76,69 @@
             <version>${workflow.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock</artifactId>
+            <version>1.57</version>
+
+            <classifier>standalone</classifier>
+            <exclusions>
```

Comment:
Mm, when you have this many `exclusions` it is a bad sign. Maybe it would be better to pick a different `classifier` (or primary artifact) so that we can pull in only the deps we really need?